### PR TITLE
Change ResourcesLoader.load signature

### PR DESCRIPTION
**Pull request by mathieuanthoine**

### CODE REVIEW

File: src/pixi/loaders/ResourceLoader.hx [hunk 1]
```diff
@@ -130,7 +130,7 @@ extern class ResourceLoader extends EventEmitter {
 	 * @param [callback] {function} Optional callback that will be bound to the `complete` event.
 	 * @return {Loader}
 	 */
-	function load(?cb:Dynamic):ResourceLoader;
```
Comment (adireddy) [inline src/pixi/loaders/ResourceLoader.hx:133]: callback can't be `Void->Void` as it can have 2 parameters. Loader itself and resources object.

Comment (adireddy) [inline src/pixi/loaders/ResourceLoader.hx:133]: better to `@:overload` if you want flexibility

File: src/pixi/core/textures/Texture.hx [hunk 2]
```diff
@@ -159,11 +159,11 @@ extern class Texture extends EventEmitter {
 	 * Helper function that creates a new Texture based on the given video element.
 	 *
 	 * @static
-	 * @param video {VideoElement}
+	 * @param video {String}
 	 * @param scaleMode {Int} See {{#crossLink "PIXI/scaleModes:property"}}scaleModes{{/crossLink}} for possible values
 	 * @return {Texture} A Texture
 	 */
-	static function fromVideo(video:VideoElement, ?scaleMode:Int):Texture;
+	static function fromVideo(video:String, ?scaleMode:Int):Texture;
```
Comment (adireddy) [inline src/pixi/core/textures/Texture.hx:166]: video can be `String` or `VideoElement` so better to use `EitherType` or `Dynamic`

https://github.com/pixijs/pixi.js/blob/master/src/core/textures/Texture.js#L402

Comment (adireddy) [reply src/pixi/core/textures/Texture.hx:166]: or even better if you can use `fromVideoUrl` method

`fromVideo` is any way calling `fromVideoUrl` if the parameter is a string

**Comment by adireddy** [comment]
Updated dev with the changes I mentioned. Let me know if it works for you.

**Comment by mathieuanthoine** [comment]
Thank you Adi,

Alas there's still too many bugs in Pixi v4 so I came back to v3 :(

**Pull request closed without merging**